src/templates/base/2019/index.html

```diff
@@ -43,7 +43,100 @@ <h2>{{ self.intro_sub_title() }}</h2>
       <img src="/static/images/home-hero.png" alt="" width="562" height="820">
     </div>
   </section>
-  {% include "%s/2019/featured_chapters.html" % lang %}
+  {% if featured_quotes %}
+  {# Get a random quote server side for those without JS #}
+  {%- set random_chapter = featured_quotes.keys()|list()|random %}
+  {%- set featured_chapter_quotes = featured_quotes.get(random_chapter) %}
+  {%- if not featured_chapter_quotes.get('stat_1') %}{% set stat1_hidden = "hidden" %}{% endif %}
+  {%- if not featured_chapter_quotes.get('stat_2') %}{% set stat2_hidden = "hidden" %}{% endif %}
+  {%- if not featured_chapter_quotes.get('stat_3') %}{% set stat3_hidden = "hidden" %}{% endif %}
+  {% if featured_chapter_quotes %}
+  <script nonce="{{ csp_nonce() }}">
+    // Get a random quote using JS for those reloading cached CDN page.
+    // We do as much as possible before HTML so we can replace the server
+    // generated quote quickly without a flash of the server side quote
```

Comment:
I'm not sold on this. The contributors page was one thing because I can see the value of giving each contributor equal prominence. But a chapter can be featured for a few hours at a time and I don't think there's any expectation from readers or authors that this should randomly update on refresh.

The overhead in both the HTML payload and the JS code just don't seem worth it for a feature that would be used infrequently and adds little to the UX.

If you're still convinced that this is the right change, could we continue this discussion in another issue and decouple it from the JSON change in this PR?

Reply:
Yeah you’re probably right. Having a chapter “featured” for a few hours is more expected. Was so preoccupied with whether or not I could that they didn't stop to think if I should 😁

Given that, should we even move to JSON or just stick with the Jinja2 template instead? We could still dynamically generate that with an EJS template like we do for the ebook. It would be an extra EJS template but as long as we use the .ejs.html naming convention we use for ebook that should be fine. Or do you think we will ever use these quotes elsewhere where having them in JSON format would be helpful?